tests/common.go

```diff
@@ -28,6 +29,8 @@ func InstallChipTool(t *testing.T) {
 	}
 	t.Cleanup(func() {
 		utils.SnapRemove(t, chipToolSnap)
+		logs := utils.SnapLogs(t, start, chipToolSnap)
+		utils.WriteLogFile(t, chipToolSnap, logs)
```

Comment:
Chip Tool snap has no services and there are no hooks that run after installation. Is there a case where there would be snap logs to dump? In any case, it's okay to keep this.

Reply:
No, these are redundant as the logs we are interested in are printed to stdout. I included dumping the journal logs for completeness.

Reply:
AppArmor denials will be listed in this log, and not in the stdout logs captured elsewhere.